recipes/minizip-ng/all/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Are you sure it's required? symbols should be properly exported: https://github.com/zlib-ng/minizip-ng/blob/1593060ecf093411f1f20f655e50992913c6d5b9/mz.h#L150-L154

Reply:
There's no `__declspec(dllimport)`.
Is this a private or public header?

Reply:
__declspec(dllimport) is not mandatory, except for global symbols.

Reply:
It does not work. Only exports DLL file.

Reply:
_WINDOWS macro should be replaced by _WIN32, that's the reason I think.

Reply:
Ok, MZ_EXPORT is not put in front of many functions of the API, so go with CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS

Reply:
Yes, I saw it requires more than a simple change. I would prefer sending it to upstream instead.